src/main/resources/alma/fix/relatedRessourcesAndLinks.fix

```diff
@@ -392,6 +392,21 @@ if exists("@inNZ")
   add_field("inCollection[].$last.label", "hbz - Hochschulbibliothekszentrum des Landes Nordrhein-Westfalen, Netzwerkzone")
 end
 
+# digiBib hbz Vk
+
+if any_equal("MBD  .M", "49HBZ_NETWORK")
+  if any_match("POR  .A", ".*")
+    add_field("@inDigibibVK", "true")
+  else
+    add_field("@inDigibibVK", "true")
+  end
+end
+
+if exists("@inDigibibVK")
```

Comment:
Why so complicated? You could simply drop this and move the `add_field` instructions up directly below line 380. Or at least get rid of the variable? Or at the _very least_ get rid of the inner conditional?